docs/CLI.md

```diff
@@ -153,7 +153,7 @@ itself.  The --trust command-line option, in conjunction with --pubkeys and
 --role, can be used to indicate the trusted keys of a role.
 
 ```Bash
-$ repo.py --trust --pubkeys --role
+$ repo.py --trust --pubkeys </path/to/foo_key.pub>+ --role <rolename>
```

Comment:
```suggestion
$ repo.py --trust --pubkeys </path/to/foo_key.pub> --role <rolename>
```
Thanks for your pr @hosseinsia!
I am not sure what do you express with the `+`?

Reply:
Hey @MVrachev, by + i meant it can be more than one but I guess people also get the point without that.

Reply:
I see, I don't mind the `+`.
I am not a maintainer, so we would have to wait for @jku, @joshuagl, or somebody else from the maintainers to approve.